pkg/kclient/binding.go

```diff
@@ -121,7 +122,45 @@ func NewServiceBindingObject(bindingName string, bindAsFiles bool, deploymentNam
 	}
 }
 
-func mappingContainsBKS(bindableObjects []*meta.RESTMapping, bks sboApi.BindableKindsStatus) bool {
+// GetBindingServiceBinding returns a ServiceBinding from group binding.operators.coreos.com/v1alpha1
+func (c Client) GetBindingServiceBinding(name string) (bindingApi.ServiceBinding, error) {
+	if c.DynamicClient == nil {
+		return bindingApi.ServiceBinding{}, nil
+	}
+
+	u, err := c.GetDynamicResource(bindingApi.GroupVersionResource, name)
+	if err != nil {
+		return bindingApi.ServiceBinding{}, err
+	}
+
+	var result bindingApi.ServiceBinding
```

Comment:
Since this would create a zero value `binding.ServiceBinding`, would declaring it at the beginning of the method and then using it in the above two cases when `DynamicClient` is `nil` or when there's error in `c.GetDynamicResource` call be simpler?

Reply:
I'm not sure. For the return with err, if I read, 
```
return result, err
```

I have to pass several seconds to understand what is the value of result.

With `return bindingApi.ServiceBinding{}, err`, I'm sure instantly it is the zero value